include/openenclave/attestation/sgx/verifier.h

```diff
@@ -25,31 +25,6 @@ OE_EXTERNC_BEGIN
  */
 oe_verifier_t* oe_sgx_plugin_verifier(void);
```

Comment:
This function is no longer defined.

Reply:
Yes this file will be removed, and a new function oe_result_t oe_verifier_initialize(void) will be defined in include/openenclave/attestation/verifier.h